app/workers/billing_worker.rb

```diff
@@ -2,8 +2,20 @@
 
 class BillingWorker
   include Sidekiq::Worker
+  include ThreeScale::SidekiqLockWorker
 
-  sidekiq_options queue: :billing, retry: 3
```

Comment:
The retries were not actually being used in `BillingWorker` before, since we catch most of the exceptions abstracting them into an element of `Finance::BillingStrategy::Results`
https://github.com/3scale/porta/blob/e4e69790516bf223fee705887c3650a1e6710ee3/app/models/finance/billing_strategy.rb#L368-L381